src/constraint/spec.ts

```diff
@@ -219,11 +219,13 @@ export const SPEC_CONSTRAINTS: SpecConstraintModel[] = [
           return specM.channelUsed(Channel.TEXT);
         case Mark.BAR:
         case Mark.CIRCLE:
-        case Mark.POINT:
         case Mark.SQUARE:
         case Mark.TICK:
         case Mark.RULE:
           return specM.channelUsed(Channel.X) || specM.channelUsed(Channel.Y);
+        case Mark.POINT:
+          return isEnumSpec(specM.enumSpecIndex.mark) ? true :
```

Comment:
Do you need to call `isEnumSpec(specM.enumSpecIndex.mark)`  here?

Isn't `specM.enumSpecIndex.mark` sufficient?